Gemfile

```diff
@@ -1,31 +1,31 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.1.3"
+ruby '3.1.3'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221355029-e50e2144-3efe-4b0a-a447-a868f6906029.png)

- [ ] Kindly add the following commands in the sub-section to make your readme file descriptive and professional :+1:

## Install

```
bundle install
rails db:create
rails db:migrate
```

Reply:
thanks.